lib/devices.js

```diff
@@ -19,9 +19,8 @@ export function waitForDeviceInfo() {
     });
 
     const cameras = devices.filter(item => item.kind === 'videoInput');
-    const microphones = devices.filter(item => item.kind === 'audioInput');
+    const microphones = devices.filter(item => item.kind === 'audioInput' && item.deviceId === 'default');//check if `headphones
```

Comment:
This will always return only one device, is this intentional? And what is the intention behind this?

Reply:
Yeah, on android devices with several microphones (e.g. with 3.5 jack) by default can be used not an existing microphone. So now, there will be always a default one.